lib/dentaku/tokenizer.rb

```diff
@@ -66,7 +66,7 @@ def scan(parent_category=nil)
         [:whitespace]
       elsif match /\/\*[^*]*\*+(?:[^*\/][^*]*\*+)*\//
         [:comment]
-      elsif match /#{numeric}\.\.#{numeric}/
+      elsif match /#{numeric}\s*\.\.\s*#{numeric}/
```

Comment:
Do you imagine that we will add something like `prettier` to control the variation? I would get pretty tripped up seeing "(1  ...         100)" as valid syntax. Would almost prefer that to throw a parse error. 

(edit: github markdown strips out the spaces but i'm just exaggerating by showing like ten spaces before and after the `...`)

Reply:
I think that is the kind of thing we need to eventually be able to trust config to handle on their own. `1 ... 100` is still a parse error because it's 3 dots instead of 1.

Reply:
There are very valid reasons for making it varied even within the same case statement, for example

```
case thing
when 1    .. 1000 then (...)
when 1001 .. 2000 then (...)
else (...)
end
```

Reply:
Interesting - thanks for that example, makes sense. We'll have to think about the process for making changes to the language (proposal -> cost/benefit ROI -> schedule -> build -> release).